sources/FILEIO

```diff
@@ -1221,6 +1221,33 @@ Copyright (c) 1981-1993, 1999, 2020-2021 by Venue & Xerox Corporation.
 (* "END EXPORTED DEFINITIONS")
 
 )
+(DEFINEQ
+
+(MAKE-EXTERNALFORMAT
+  [LAMBDA (NAME INCCODEFN PEEKCCODEFN BACKCCODEFN OUTCHARFN FORMATBYTESTREAMFN EOL)
+                                                            (* ; "Edited  1-Aug-2021 23:13 by rmk:")
+
+    (* ;; "Compiled creator for EXTERNALFORMAT so that declaration (EXPORTS.ALL) is not needed.  If EOL is not specified, then EOLVALID is also NIL")
```

Comment:
Many of the lower-level things that the format implementations make use of are actually function calls (\BOUT, \BIN). This means that formats that stay within that scope don't have to be dependent on EXPORTS.ALL.  It may or may not be useful.